nlpiper/transformers/embeddings.py

```diff
@@ -12,23 +15,49 @@
 from nlpiper.logger import log
 
 __all__ = [
-    "GensimEmbeddings"
+    "GensimEmbeddings",
+    "TorchTextEmbeddings"
 ]
 
 
 class GensimEmbeddings(BaseTransformer):
-    """Gensim Embedding extraction."""
+    """Gensim Embedding extraction.
 
-    def __init__(self, keyed_vectors: Any, apply_doc: str = 'mean'):
-        """Stem tokens.
+    Callable arguments:
+
+    Args:
+            keyed_vectors (Any): Gensim model based on keyedVectors,
+                see more in: https://radimrehurek.com/gensim/models/keyedvectors.html
+            apply_doc (str): Mode to calculate the embeddings vector for the document,
+                which could be `"mean"` or `"sum"` of the tokens.
+
+    Returns: Document
+
+    Example:
+        import gensim.downloader
+
+        from nlpiper.transformers.embeddings import GensimEmbeddings
+        from nlpiper.core.document import Document
+        from nlpiper.transformers.tokenizers import BasicTokenizer
+
+        glove_vectors = gensim.downloader.load('glove-twitter-25')
+        doc = Document('Test random stuff.')
 
-        Gensim Embedding extraction.
+        t = BasicTokenizer()
+        t(doc, inplace=True)
+
+        e = GensimEmbeddings(glove_vectors, True)
```

Comment:
Is this example correct? It should create the embedding and then apply it, no?

Reply:
yeah, I forgot to apply it